splink/cluster_studio.py

```diff
@@ -228,41 +228,51 @@ def _get_cluster_id_of_each_size(
     return df_cluster_sample_with_size.as_record_dict()
 
 
-def _get_lowest_density_cluster_ids(
+def _get_lowest_density_clusters(
     linker: "Linker",
     df_cluster_metrics: SplinkDataFrame,
-    sample_size: int,
+    rows_per_partition: int,
     min_nodes: int,
 ):
-    """Retrieves cluster IDs based on density metric, ordered from least to
-    most dense.
+    """Returns ids of lowest density clusters of different sizes by
```

Comment:
think this docstring needs more updating to reflect the fact we are returning full records, not just clusters